graphql missing from package.json

```
Module not found: Can't resolve 'graphql' in '/Users/ed/Development/ethdenver-dapp/node_modules/apollo-link-state/lib'
```

adding graphql via `yarn add graphql --save` solved this issue!

--- a/package.json
+++ b/package.json
@@ -6,6 +6,7 @@
     "@material-ui/core": "^3.9.2",
     "@material-ui/icons": "^3.0.2",
     "apollo-boost": "^0.1.28",
+    "graphql": "^14.1.1",
     "react": "^16.8.2",
     "react-apollo": "^2.4.1",
     "react-dom": "^16.8.2",

